pkgs/by-name/re/remind/package.nix

```diff
@@ -1,32 +1,36 @@
-{ lib
-, stdenv
-, fetchurl
-, tk
-, tcllib
-, tcl
-, tkremind ? true
+{
+  lib,
+  stdenv,
+  fetchurl,
+  tk,
+  tcllib,
+  tcl,
+  tkremind ? true,
```

Comment:
it needs a better name.

Reply:
I thought so too, @AndersonTorres, but was hesitant as I believe it would be a breaking change for folks disabling it.

What do you think about naming it `withGUI`?
And what would be a good path forward? Is an entry in the release notes enough?

Reply:
I remember that we did an alias for Emacs options.
This should work:

```nix
tkremind ? null,
withGui ? 
if tkremind != null then 
  lib.warn "tkremind is deprecated and should be removed;  use withGui instead." tkremind 
else 
  true,
. . . 
```

Reply:
Great suggestion, thanks @AndersonTorres, I've update this PR accordingly.

Reply:
Now it has a Wiki page!

https://wiki.nixos.org/wiki/Nix_Cookbook#Deprecating_a_specific_input_parameter_in_mkDerivation-style_packages